.grabl/automation.yml

```diff
@@ -36,7 +36,6 @@ build:
     test-integration:
       image: vaticle-ubuntu-20.04
       command: |
-        bazel build //test/integration/...
```

Comment:
This line is redundant; `bazel test` builds everything it requires for testing anyway.